app/views/admin/order_cycles/_wizard_progress.html.haml

```diff
@@ -1,3 +1,18 @@
 %ul.wizard-progress
-  %li{ ng: { repeat: "step in ['general_settings','incoming','outgoing']", class: '{current: view==step}' } }
-    {{ "admin.order_cycles.wizard_progress."+ step | t }}
+  - if action_name == 'new'
+    %li{ class: "current" }
+      = t("admin.order_cycles.wizard_progress.edit")
+    %li
+      = t("admin.order_cycles.wizard_progress.incoming")
+    %li
+      = t("admin.order_cycles.wizard_progress.outgoing")
+  - else
+    %li{ class: "#{'current' if action_name == 'edit'}" }
+      %a{href: "#{main_app.edit_admin_order_cycle_path(@order_cycle)}"}
```

Comment:
Why not `%a{href: main_app.edit_admin_order_cycle_path(@order_cycle)}`? Is that due to some Haml bug?

Reply:
string interpolation was needed in a previous version of the code, now it's not. I removed it in a new commit. thanks!